platform/node/test/js/set_center.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var mockfs = require('./../mockfs');
+var mbgl = require('../../index');
+var test = require('tape');
+
+var options = {
+    request: function(req, callback) {
+        callback(null, { data: mockfs.dataForRequest(req) });
+    },
+    ratio: 1,
+    center: ["ads", "asd"]
+};
+
+test('SetCenter', function(t) {
+    t.test('sanity', function(t) {
+        var renderCallback = function(err, pixels) { if (!err) t.end(); };
+        var map = new mbgl.Map(options);
+
+        map.load(mockfs.style_vector);
+        map.setCenter([0, 0]);
+
+        map.render({ zoom: 16 }, renderCallback);
+    });
+
+    t.test('invalid arguments', function(t) {
+        var renderCallback = function(err, pixels) { if (!err) t.end(); };
```

Comment:
In the case where we have an error, we should not end it right away. Instead, let's call `t.ifError(err);` which will surface the error better.